keras_nlp/models/albert/albert_presets.py

```diff
@@ -102,10 +111,13 @@
             "num_segments": 2,
         },
         "preprocessor_config": {},
-        "description": (
-            "Extra Large size of ALBERT where all input is lowercased. "
-            "Trained on English Wikipedia + BooksCorpus."
-        ),
+        "metadata": {
+            "description": (
+                "Extra Large size of ALBERT where all input is lowercased. "
+                "Trained on English Wikipedia + BooksCorpus."
+            ),
+            "params": 222595584,
```

Comment:
We can actually add `"official_name"` and `"path"` for all models, even ones that are not yet documented on keras.io.

The `path` should always match the directory the model is in in KerasNLP. The `official_name` should match the original paper.